src/components/pages/ClassInfoPage.jsx

```diff
@@ -0,0 +1,227 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Once this component is separated into `ClassInfo` and `TeachInfo` objects like `classItems` and `teacherInfo` can be destructured to clean up some of the repetition in the code base.